javaScript.js

```diff
@@ -88,6 +88,34 @@ const projects = [
   },
 ];
 
+const form = document.getElementById('contactForm');
+const email = document.getElementById('email');
+const alert = document.getElementById('alertWin');
+const alertText = document.getElementById('alert-text');
+email.addEventListener = ('keyup', () => { alert.style.display = 'none'; });
+
+const getInfo = () => {
+  const formData = JSON.parse(localStorage.getItem('formData'));
+
+  if (formData) {
+    document.getElementById('full-name').value = formData.fullname;
+    document.getElementById('email').value = formData.email;
+    document.getElementById('message').value = formData.message;
+  }
+};
+
+form.addEventListener('keyup', () => {
+  const formData = {
+    fullname: document.getElementById('full-name').value,
+    email: document.getElementById('email').value,
+    message: document.getElementById('message').value,
+  };
+
+  localStorage.setItem('formData', JSON.stringify(formData));
+}, true);
+
+getInfo();
```

Comment:
- [ ] After filling the form, and not submitting it, the entered values should be saved in the browser but it is not saving so kindly make a change and implement it.